Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM node:14-alpine
 ENV NODE_OPTIONS=--max-old-space-size=8192
 
 # Install build dependancies.
-RUN apk --no-cache add git python3
+RUN apk --no-cache add git python3 gcc g++ libc-dev musl musl-dev make
```

Comment:
adding these allows python to run beneath node to build/retrieve the tools necessary to build/run nodejs and other dependencies.